frontend/contact/index.html

```diff
@@ -125,6 +123,21 @@ <h3>My domain (or one of its subdomains) is broken due to preloading, and waitin
 
   <section id="other">
     <h2><a class="hash-link" href="#other">Other questions</a></h2>
+
+    <h3>Customer support for my service provider told me to contact you for support</h3>
+    <p>
+      We apologize that your service provider instructed you to contact us, as we are unable to assist you.
```

Comment:
Consider: "Unfortunately, we're unable to provide individual sites with help troubleshooting their HSTS configuration."

Reply:
Done, and added another sentence.